server/athenian/api/controllers/selector.py

```diff
@@ -0,0 +1,80 @@
+from typing import Callable, Optional
```

Comment:
Let's name it `feature_selector` or `calculator_selector`. My first association was UNIX's `select()` :smile:

Reply:
haha 👍